operator/src/main/resources/scripts/modelInImage.sh

```diff
@@ -260,9 +261,66 @@ function buildWDTParams_MD5() {
     OPSS_FLAGS=""
   fi
 
+  overrideWDTTimeoutValues
+
   trace "Exiting setupInventoryList"
 }
 
+function changeTimeoutProperty() {
+  trace "Entering changeTimeoutProperty"
```

Comment:
This tracing seems pretty verbose for such a low level function...

Reply:
done